src/DynamoCore/Core/DynamoMigrator.cs

```diff
@@ -302,24 +326,23 @@ internal static DynamoMigratorBase Migrate(IPathResolver pathResolver, FileVersi
         }
 
         /// <summary>
-        /// Given a root user directory, this creates an instance of the migrator
+        /// Given a FileVersion, this creates an instance of the migrator
         /// depending on the input version using reflection. Returns the default
         /// migrator (DynamoMigratorBase) if version specific migrator does not exist
         /// </summary>
-        /// <param name="pathResolver"></param>
         /// <param name="version"> input version for which migrator instance is created </param>
         /// <returns> instance of migrator specific to input version. </returns>
-        internal static DynamoMigratorBase CreateMigrator(IPathResolver pathResolver, FileVersion version)
+        internal static DynamoMigratorBase CreateMigrator(FileVersion version)
         {
             var className = "Dynamo.Core.DynamoMigrator" + version.MajorPart + version.MinorPart;
 
-            var args = new object[] { pathResolver, version };
+            var args = new object[] { version };
             var type = Assembly.GetExecutingAssembly().GetType(className);
 
             if(type != null)
                 return (DynamoMigratorBase)Activator.CreateInstance(type, args);
```

Comment:
Since this `if` statement doesn't conform to the coding standard, we might as well move the `args` inside the statement so it looks this way:

```csharp
if (type != null)
{
    var args = new object[] { version };
    return (DynamoMigratorBase)Activator.CreateInstance(type, args);
}
```